htdocs/compta/sociales/class/chargesociales.class.php

```diff
@@ -39,6 +39,11 @@ class ChargeSociales extends CommonObject
 	 */
 	public $element = 'chargesociales';
 
+	/**
+	 * @var string Name of table without prefix where object is stored
+	 * @deprecated
```

Comment:
I recommend to add "Use $...." after `@deprecated` because this shows up in the phan reports.

Reply:
is it good now ?

Reply:
Yes, that will show up when derpacation notices are enabled.